ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+## Expected Behavior
```

Comment:
I've seen it really helpful to add HTML comments to give more details for example, something like `<!-- sometimes, one person's bug is another person's feature. It's really helpful to know not only what unexpected behavior you experienced, but also what behavior you expected -->`